src/encoded/static/components/regulome_search.js

```diff
@@ -5,16 +5,20 @@ import url from 'url';
 import * as globals from './globals';
 import { SortTablePanel, SortTable } from './sorttable';
 import { Motifs } from './motifs';
-import { BarChart, ChartList, ChartTable, lookupChromatinNames } from './visualizations';
-import { requestSearch, shadeOverflowOnScroll } from './objectutils';
+import { BarChart, ChartList, lookupChromatinNames } from './visualizations';
+import { requestSearch } from './objectutils';
 import GenomeBrowser from './genome_browser';
 import { Modal, ModalHeader, ModalBody, ModalFooter } from '../libs/bootstrap/modal';
+import { FacetList, filterByAllSelectedFilters } from './facets';
+import { ChromatinView } from './chromatin_view';
 
 const screenMediumMax = 787;
 const screenSmallMax = 483;
 
-// Number of terms to show, the rest will be viewable on scroll
-const displayedTermsCount = 14;
+// Define facets (probably this should be in a schema really)
+const facetList = ['file_format', 'organ', 'biosample', 'assay', 'target'];
+const facetTitleList = ['File format', 'Organ / cell type', 'Biosample', 'Method', 'Target'];
+const typeaheadFacetList = [false, true, true, false, true];
```

Comment:
Would it be more straightfoward to implement these as one array of objects, to make it easier to keep these in sync? I realize this is from existing code.